addons/website/static/src/js/content/snippets.animation.js

```diff
@@ -551,6 +557,27 @@ registry.slider = Animation.extend({
         });
     },
 
+    /**
+     * @private
+     */
+    _activateNextOrDeleteCarousel: function () {
+        var self = this;
+        var $next = self.$target.find('.carousel-item:not(.active)');
+
+        if ($next.length) {
+            $next.first().addClass('active');
+            this.$target.find('ol.carousel-indicators li').last().remove();
+        } else {
+            this.destroy();
```

Comment:
@qsm-odoo Is calling `destroy` necessary before removing the snippet from the DOM? (If yes, it should probably be moved in the async call...)